src/D2L.Security.OAuth2.WebApi/D2L.Security.OAuth2.WebApi.csproj

```diff
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	<Sdk Name="Microsoft.Build.CentralPackageVersions" Version="2.0.1" />

```

Comment:
You can also centralize the version of this with `global.json`:

https://search.d2l.dev/xref/lms/global.json?r=66ab99aa

Reply:
Oh nice, will do.